### DISCUSSION TITLE
SIWS support via deeplinks

### DISCUSSION THREAD

**Discussion by Mattijah** [phantom/docs#247] [General]
Hey,

Looking at the docs, it seems SIWS (sign in with solana) is not supported via deeplinks? Is that right? I know I can still use `signMessage` and get around it, but the form will say "_Sign Message_" instead of "_Sign In_". Would be nice to have this supported (if it's not). If it is, where can I find relevant docs?

https://docs.phantom.app/phantom-deeplinks/provider-methods

Thank you

**Comment by adamdelphantom** [comment]
Hello @Mattijah,

Yes that is correct. SIWS is not yet supported with deep links. In addition to this comment please consider leaving a feature request [here](http://phantom.canny.io). Where others can upvote your request to help with consideration.

**Reply by mooshee** [reply]
Can we re-open this request or get an update? Apparently this is not a "feature request". 

https://phantom.canny.io/feature-requests/p/signin-support-via-deeplink
